server/api/shiftView.js

```diff
@@ -0,0 +1,178 @@
+const dbClient = require('./dbClient')
+const { verifyJWTAuthToken } = require('./dataHelper')
+
+const shiftViewGETCallback = async (req, res) => {
+  try {
+    await verifyJWTAuthToken(req, res)
+
+    const shiftViewData = req.body
+
+    if (!shiftViewData.date || !shiftViewData.positionId) {
+      throw new Error('Missing/Invalid', { cause: { error: { status: 400, message: 'Missing Required Data' } } })
+    }
+
+    // slices off the time
+    shiftViewData.date = shiftViewData.date.slice(0, 10)
+
+    const timeBlocks = await getTimeBlocks(shiftViewData)
+    const employees = await getEmployees(shiftViewData)
+
+    const formattedEmployees = []
+
+    for (let i = 0; i < employees.length; i++) {
+      const employeeInfo = {
+        employeeId: employees[i].userId,
+        employeeName: employees[i].userName
+      }
```

Comment:
For readability this can be changed to a for of loop
i.e. for (const employee of employees)